build/azure-pipelines.yml

```diff
@@ -12,7 +12,7 @@ variables:
 - name: NUGET_VERSION
   value: 6.2.0
 - name: VSTEST_PLATFORM_VERSION
-  value: 17.2.5
+  value: 17.4.0
```

Comment:
Is 17.4.0 necessary for the VSTEST? 17.2.5 is still used for the windows container

Reply:
1https://www.nuget.org/packages/Microsoft.TestPlatform/17.1.0-release-20220113-05 it doesnt exist, or IDK, the pipeline couldnt get it. The only time this task succeeded before that change, it happened because another version was already in the cache and it used it instead of 17.2.5 after it failed to get it.

Reply:
If it's supposed to exist, I don't know if we're supposed to specify we get the nuget from somewhere else?

Reply:
I would personally align the TestPlatform with the container being used which is 17.2.5.